Formula/syncthing.rb

```diff
@@ -17,17 +17,17 @@ class Syncthing < Formula
   depends_on :hg => :build
 
   def install
-    ENV["GOPATH"] = cached_download/".gopath"
+    ENV["GOPATH"] = cached_download
```

Comment:
Any idea why this wants built in the download path? Would be really nice to avoid that if we can.